Freshli.Web/Views/AnalysisRequests/Create.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model AnalysisRequest
+@{
+    ViewData["Title"] = "Home";
+}
+
+<div>
+    <form asp-route="CreateAnalysisRequest" method="post">
+        <div asp-validation-summary="All" class="text-danger"></div>
```

Comment:
This line is the only thing that changed between the old `Index` view and this view.